README.md

```diff
@@ -128,7 +123,7 @@ id      name    active
 2       bar     false
 ```
 
-With `-t` option, results are displayed in table format
+With `-t` option, results are displayed in table format.
 
 ```
 $ spanner-cli -p myproject -i myinstance -d mydb -e 'SELECT * FROM users;' -t
```

Comment:
Sounds good! Cloud Spanner doesn't support `TRUNCATE Table` query, but it would be possible if we convert it to the `DELETE FROM table_name WHERE true` query.